api/http/api_deployments.go

```diff
@@ -965,8 +990,19 @@ func parseEpochToTimestamp(epoch string) (time.Time, error) {
 func (d *DeploymentsApiHandlers) LookupDeployment(w rest.ResponseWriter, r *rest.Request) {
 	ctx := r.Context()
 	l := requestlog.GetRequestLogger(r)
+	q := r.URL.Query()
+	defer func() {
+		var redacted bool
+		if search := q.Get("search"); search != "" {
+			q.Set("search", "REDACTED")
+			redacted = true
```

Comment:
why do we need the flag `redacted` here?

Reply:
We don't :)